Sources/SwifterSwift/Combine/PublisherExtensions.swift

```diff
@@ -0,0 +1,123 @@
+//
+//  PublisherExtensions.swift
+//  SwifterSwift
+//
+//  Created by Heecheon Park on 7/28/23.
+//  Copyright © 2023 SwifterSwift
+//
+
+// swiftlint:disable large_tuple
+#if canImport(Foundation) && canImport(Combine)
+import Foundation
```

Comment:
Do you need `Foundation`?
```suggestion
#if canImport(Combine)
```